addons/l10n_in_website_sale/models/res_config_settings.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = ['res.config.settings']
+
+    auto_check_gstin_status_website = fields.Boolean(
```

Comment:
```suggestion
    l10n_in_auto_check_gstin_status_website = fields.Boolean(
```

Reply:
```suggestion
```
Remove this field and add `module_l10n_in_website_sale` in l10n_in `res.config.settings`